entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/EntityViewManagerImpl.java

```diff
@@ -1450,6 +1450,11 @@ public EntityViewManager getSerializableDelegate(Class<?> entityViewClass) {
         return serializableDelegates.get(proxyFactory.getProxy(this, metamodel.managedViewOrError(entityViewClass)));
     }
 
+    @Override
+    public void close() {
+        proxyFactory.clear();
```

Comment:
I think we should add a `static volatile boolean closed` as well and set this here, as well as check on every public method if it is closed to throw an exception when that is the case.
Apart from that, do you know if the static implementations (generated by the annotation processor) are "unloaded" properly as well?

Reply:
> I think we should add a static volatile boolean closed as well and set this here, as well as check on every public method if it is closed to throw an exception when that is the case.

Why would we do that? What's the point? The contract of the close method clearly states that any invocations on an entity view manager that has been closed is undefined.

> Apart from that, do you know if the static implementations (generated by the annotation processor) are "unloaded" properly as well?

No, I need to check that.

Reply:
>Why would we do that? What's the point? The contract of the close method clearly states that any invocations on an entity view manager that has been closed is undefined.

Yes, but undefined is not very user friendly. If we get into some state when bad things might happen after closing, then we should ensure a user is not using the EVM anymore after a close. I created https://github.com/Blazebit/blaze-persistence/issues/1813 for this task as I don't want to hold back the release for this any further than necessary, but if you find time to work on that, I'd very much appreciate it.

Reply:
> Apart from that, do you know if the static implementations (generated by the annotation processor) are "unloaded" properly as well?

Judging from `com.blazebit.persistence.view.impl.proxy.ProxyFactory#loadImplementation` static implementations should be handled properly as well by this change.